cdap-common/src/main/java/io/cdap/cdap/common/utils/EstablishHealthCheckService.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright © 2014 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.common.utils;
+
+import io.cdap.cdap.common.conf.CConfiguration;
+import io.cdap.cdap.common.conf.Constants;
+import io.cdap.cdap.common.service.HealthCheckService;
+
+/**
+ * General start specific health check service.
+ */
+public class EstablishHealthCheckService {
+  public void startHealthCheckServiceByServiceName(HealthCheckService healthCheckService, String serviceName,
```

Comment:
nit: Can probably make this a static helper method in the HealthCheckService class.